lib/tool_belt/cherry_picker.rb

```diff
@@ -27,7 +27,7 @@ def find_cherry_picks(project, release, repo_names)
         commits = issue['changesets']
 
         commits.each do |commit|
-          if !commit['comments'].start_with?('Merge pull request') && !@release_environment.commit_in_release_branch?(repo_names, commit['comments'])
+          if commit['comments'].downcase.start_with?('fixes', 'refs') && !@release_environment.commit_in_release_branch?(repo_names, commit['comments'])
```

Comment:
During my testing I ran into instances where there were commits that had neither refs nor fixes and they appeared to be commits that should not be cherry-picked.